src/bdscript/parentID.md

```diff
@@ -0,0 +1,36 @@
+# $parentID
+Returns the parent category ID of a channel.
+
+## Usages
+There are two usages of this function :
+
+### Usage #1
+#### Syntax
+```
+$parentID
+```
+Returns the parent category ID of the current channel.
+
+#### Example
+```
+$nomention
+Current category: $parentID
+```
+![example](https://user-images.githubusercontent.com/94063167/199015904-8c1635fc-ae14-40e6-9102-5f0a94a65cb7.png)
+
+### Usage #2
+#### Syntax
+```
+$parentID[Channel Name]
```

Comment:
`$parentID[]` accepts a channel ID, not a name.
Since we already have functions to get channel IDs from names, I decided to make `$parentID[]` take ID instead of a name.

Please update the rest of the page accordingly.